charts/simple-app/templates/deployment.yaml

```diff
@@ -203,11 +203,11 @@ spec:
           {{- end }}
           {{- with $.Values.livenessProbe }}
           livenessProbe:
-            {{- toYaml . | nindent 12 }}
+            {{- toYaml tpl (required "livenessProbe is required" .) $ | nindent 12 }}
```

Comment:
The k8s onboarding experience is currently not great due to this default setting in particular. I don't know why this would be any different/more confusing than all the other places that use `required`

Reply:
Fixed by adding required in documentation for values.yaml file